test/studies/hpcc/HPL/vass/hpl.hpcc2012.ml-perf.graph

```diff
@@ -1,5 +0,0 @@
-perfkeys: Performance (Gflop/s) =
```

Comment:
You'll need to remove the .graph files from $CHPL_HOME/test/ML-GRAPHFILES too

Reply:
I was wondering about this -- what happens if I don't?  Do we just stop getting data after that point, or do we get an error / failure of some sort?  (I was hoping the former...)

Reply:
start_test checks that all .graph files are listed in a *GRAPHFILES file and vice-versa. It'll issue a warning if that's not true. (We wanted to split out this check and run it as part of the smoke test https://github.com/chapel-lang/chapel/issues/6422 ) If you want to keep the graph around, just leave the .graph files in.

Reply:
FYI I think `start_test --performance test/studies/hpcc/HPL/vass/` should have issued the warning

Reply:
Thanks for the clarification / reminder.  (I got dozens of warnings on that run, so must've missed it in the noise.  But... the normal `-performance` run would check the ML-GRAPHFILES as well?)

Reply:
Yeah, it should. We just look for *GRAPHFILES and *.graph so it should catch all perf configs

Reply:
OK, didn't anticipate that, thanks.

Reply:
Do you not get dozens of warnings about studies/prk/* .graph files being missing from GRAPHFILES when you run start_test -performance on a specific subdirectory/test?  I can't think what I'd have done personally to trigger that.

Reply:
Hmm, are you on a case-insensitive system (mac)?

#6405 changed the case of some prk tests. I'm wondering if on a case-insensitive system if the git pull of #6405 didn't change the names locally. I would try doing `rm -r $CHPL_HOME/test/studies/prk && git co $CHPL_HOME/test/sudies/prk`